src/Paket.Core/Installation/BindingRedirects.fs

```diff
@@ -83,17 +83,6 @@ let internal indentAssemblyBindings config =
         if not parent.HasElements then
             parent.Remove()
 
-let private configFiles = [ "app"; "web" ] |> Set.ofList
-let private projectFiles = [ ".csproj"; ".vbproj"; ".fsproj"; ".wixproj"; ".nproj"; ".vcxproj"; ".pyproj"; ".sfproj"; ".sqlproj" ] |> Set.ofList
```

Comment:
I think we need to keep this ugly explicit list

Reply:
What for? It's private, not used anywhere, and is just a duplicating the list in ProjectFile.fs

Reply:
wasn't this was filtered things to  "*.*proj"?

Reply:
No, not since https://github.com/fsprojects/Paket/commit/aac9770c6bc5e11261703685a5f15a6fec937b21